Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -209,6 +209,7 @@ def GetPlatformDsc (self):
             'GpioLib|Silicon/$(SILICON_PKG_NAME)/Library/GpioLib/GpioLib.inf',
             'SpiFlashLib|Silicon/$(SILICON_PKG_NAME)/Library/SpiFlashLib/SpiFlashLib.inf',
             'GraphicsInitLib|Silicon/CommonSocPkg/Library/EmuGfxInitLib/EmuGfxInitLib.inf',
+            'BootGuardLib|Silicon/CommonSocPkg/Library/BootGuardLibCBnT/BootGuardLibCBnT.inf',
```

Comment:
I would recommend to link the library which APL uses. We do not checkAcmsvn support for this lib and QEMU can reutilize it. Ideally, we should give NULL library as QEMU doesn't support boot guard,